<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calendar_day</class>
 <widget class="QWidget" name="Calendar_day">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>198</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,1,1,2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="dateLabel">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>28</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label1">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 8pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>事项1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label2">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 8pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>事项2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label3">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 8pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>事项3</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
